shell/platform/linux/config/BUILD.gn

```diff
@@ -12,7 +12,10 @@ if (build_glfw_shell) {
 }
 
 pkg_config("gtk") {
-  packages = [ "gtk+-3.0" ]
+  packages = [
+    "gtk+-3.0",
+    "wayland-client",
```

Comment:
wayland-client should not be part of this config, it should be its own config which the relevant target includes.